packages/tools/src/utilities/math/polyline/getSignedArea.ts

```diff
@@ -5,9 +5,9 @@ import type { Types } from '@cornerstonejs/core';
  * https://www.youtube.com/watch?v=GpsKrAipXm8&t=1900s
  *
  * This functions has a runtime very close to `getArea` and it is recommended to
- * be called only if you need the area signal (eg: calculate polygon normal). If
- * you do not need the area signal you should always call `getArea`.
- *
+ * be called only if you need the area signal (eg: calculate polygon normal or
+ * winding direction). If you do not need the area signal you should always call
+ * `getArea`.
```

Comment:
I'm not sure about the performance of the getSignedArea now, but if you want a bit more performance, then passing in a TypedArray<Types.Point2> and directly accessing the array data will be faster since you avoid creating so many tiny arrays.  Again, not part of this PR, but something to consider for the future.